src/lxc/conf.c

```diff
@@ -4037,6 +4147,27 @@ int lxc_clear_cgroups(struct lxc_conf *c, const char *key)
 	return 0;
 }
 
+int lxc_clear_limits(struct lxc_conf *c, const char *key)
+{
+	struct lxc_list *it, *next;
+	bool all = false;
+	const char *k = key + 10;
```

Comment:
Yeah could make it less magical (note the access to `k` later depends on `all` being `false`, which is an easily overlooked dependency, and error prone). I should probably also change `lxc_clear_cgroups` where I took the pattern from ;-).